lib/hz2600/Kazoo/Api/Collection/Transactions.php

```diff
@@ -0,0 +1,12 @@
+<?php 
+
+namespace Kazoo\Api\Collection;
+
+class Transactions extends AbstractCollection
+{
+    public function current_balance() {
```

Comment:
for uniformity this should be called currentBalance. Only the property setter functions in the call flow builder use snake_case. For the entity classes we use camelCase for function names.